G1ANT.Addon.Selenium/Api/SeleniumIFrameArguments.cs

```diff
@@ -0,0 +1,13 @@
+﻿using G1ANT.Language;
+
+namespace G1ANT.Addon.Selenium
+{
+    public class SeleniumIFrameArguments : CommandArguments
```

Comment:
you should change SeleniumCommandArguments base class to SeleniumIFrameArguments and remove redundant properties

Reply:
zrobiłem tak kolejność jest ok. Zaraż zrobię pusha